wallet-mobile-app-centbit/Components/HomeFirstCom.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { COLORS } from '../Constants/colors';
+
+export default function HomeFirstComponent(){
+     return(
+          <View style={{
+               flexDirection: 'row',
+               justifyContent: 'space-between',
+               padding: 20,
+          }}>
+               <View>
+                    <Text style={{
+                         color: COLORS.colorDarkGray,
+                         marginBottom: 5,
+                         fontSize: 15
```

Comment:
creating a constant fontSize is better, having something like headerSize, titleSize, subTitle that way...you can easily change headerSize and it will impact all the headers